examples/backup_vm.py

```diff
@@ -270,7 +270,7 @@ def add_start_backup_args(parser):
         help="UUID of the VM to backup.")
```

Comment:
What about vm_uuid ? no this is not consistent..

Reply:
The consistency issue is with all the other optional arguments. All of them are in the form\
`--xxx-xxx`. All the positional arguments are `xxx_xxx` since there is no way to have 
positional argument using `xxx-xxx`.

This argument was just introduced and never release so fixing it will not break anyone, but
even if qe are already using this, they easily change their code.